kotlin/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
```

Comment:
I don't know much about Kotlin, but are all these "config" files like `kotlin/gradlew` or `kotlin/gradlew.bat` necessary? I assume that Gradle is the dependency manager from Java/Kotlin, but I've never committed the `composer.phar` file in the projects I've worked, instead we usually use composer form the "machine" (local/docker image).

Maybe it's just my PHP background, but I would expect having something lighter as starting point and build from there... Or is this something related to Kotlin best practices? Could you help me understanding this, @davidisla? 😄

Reply:
All right, thanks for the answer 🙌